src/_data/sidenav.yml

```
@@ -403,31 +403,32 @@
     - title: Testing overview
       permalink: /testing/overview
       match-page-url-exactly: true
-    - title: Widget testing
       children:
-        - title: Introduction
-          permalink: /cookbook/testing/widget/introduction
-        - title: Find widgets
-          permalink: /cookbook/testing/widget/finders
-        - title: Simulate scrolling
-          permalink: /cookbook/testing/widget/scrolling
-        - title: Simulate user interaction
-          permalink: /cookbook/testing/widget/tap-drag
-    - title: Integration testing
-      children:
-        - title: Introduction
-          permalink: /testing/integration-tests
-          match-page-url-exactly: true
-        - title: Write and run an integration test
-          permalink: /cookbook/testing/integration/introduction
-        - title: Profile an integration test
-          permalink: /cookbook/testing/integration/profiling
-    - title: Unit testing
-      children:
-        - title: Introduction
-          permalink: /cookbook/testing/unit/introduction
-        - title: Mock dependencies
-          permalink: /cookbook/testing/unit/mocking
+        - title: Widget testing
+          children:
+            - title: Introduction
+              permalink: /cookbook/testing/widget/introduction
+            - title: Find widgets
+              permalink: /cookbook/testing/widget/finders
+            - title: Simulate scrolling
+              permalink: /cookbook/testing/widget/scrolling
+            - title: Simulate user interaction
+              permalink: /cookbook/testing/widget/tap-drag
+        - title: Integration testing
+          children:
+            - title: Introduction
+              permalink: /cookbook/testing/integration/introduction
+              match-page-url-exactly: true
+            - title: Write and run an integration test
+              permalink: /testing/integration-tests
```

Thanks for swapping these! Taking a closer look however, I remember now that there is still a lot of overlap between these two pages.

What do you think about removing the "/testing/integration-tests" one from the sidenav for now in this pull request, then a separate issue can be made for updating and consolidating these pages?

```suggestion
            - title: Introduction
              permalink: /cookbook/testing/integration/introduction
              match-page-url-exactly: true
```

---

I think this should remain in this PR because the issue made #9434, was made to capture this problem also, so making another issue with the same description will create confusion